e2e/e2e_test.go

```diff
@@ -257,6 +390,59 @@ func verifyTransferToBridgeAccount(setup *setup.Setup, memo string, whbarReceive
 	return *transactionResponse, whbarBalanceBefore
 }
 
+func verifyTokenTransferToBridgeAccount(setup *setup.Setup, memo string, wTokenReceiverAddress common.Address, t *testing.T) (hedera.TransactionResponse, *big.Int) {
+	// Get the wrapped hts token balance of the receiver before the transfer
+	wrappedTokenBalanceBefore, err := setup.Clients.WTokenContract.BalanceOf(&bind.CallOpts{}, wTokenReceiverAddress)
+	if err != nil {
+		fmt.Println(`Unable to query the token balance of the  receiver account`)
+		t.Fatal(err)
+	}
+
+	fmt.Printf("Token balance before transaction: [%s]\n", wrappedTokenBalanceBefore)
+	// Get bridge account token balance before transfer
+	receiverBalance, err := hedera.NewAccountBalanceQuery().
+		SetAccountID(setup.BridgeAccount).
+		Execute(setup.Clients.Hedera)
+	if err != nil {
+		fmt.Println(`Unable to query the token balance of the Bridge Account`)
+		t.Fatal(err)
+	}
+	fmt.Println(fmt.Sprintf(`Bridge account Token balance before transaction: [%d]`, receiverBalance.Token[setup.TokenID]))
+
+	// Get the transaction receipt to verify the transaction was executed
+	transactionResponse, err := sendTokensToBridgeAccount(setup, memo)
+	if err != nil {
+		fmt.Println(fmt.Sprintf(`Unable to send Tokens to Bridge Account, Error: [%s]`, err))
+		t.Fatal(err)
```

Comment:
We can merge these two lines into one: `t.Fatalf(`Unable to send Tokens to Bridge Account, Error: [%s]`, err)`